doc/files/opensc.conf.5.xml.in

```diff
@@ -0,0 +1,1630 @@
+									</para>
+								</listitem>
+								<listitem>
+									<para>
+										<filename><envar>USERPROFILE</envar>\.eid-cache\</filename> (Windows)
+									</para>
+								</listitem>
+							</itemizedlist>
+						</para>
+						<para>
+							If caching is done by a system process, the
+							cached files may be placed inaccessible from
+							a user account. Use a globally readable and
+							writable location if you wish to share the
+							cached information. Note that the cached files
+							may contain personal data such as name and mail
+							address.
+					</para></listitem>
+				</varlistentry>
+				<varlistentry>
+					<term>
+						<option>use_pin_caching = <replaceable>bool</replaceable>;</option>
+					</term>
+					<listitem><para>
+							Use PIN caching (Default: <literal>true</literal>)?
+					</para></listitem>
+				</varlistentry>
+				<varlistentry>
+					<term>
+						<option>pin_cache_counter = <replaceable>num</replaceable>;</option>
+					</term>
+					<listitem><para>
+							How many times to use a PIN from cache before
+							re-authenticating it (Default:
+							<literal>10</literal>)?
+					</para></listitem>
+				</varlistentry>
+				<varlistentry>
+					<term>
+						<option>pin_cache_ignore_user_consent = <replaceable>bool</replaceable>;</option>
+					</term>
+					<listitem><para>
+							Older PKCS#11 applications not supporting
+							<literal>CKA_ALWAYS_AUTHENTICATE</literal> may
+							need to set this to get signatures to work with
+							some cards (Default: <literal>false</literal>).
+					</para></listitem>
+				</varlistentry>
+				<varlistentry>
+					<term>
+						<option>enable_pkcs15_emulation = <replaceable>bool</replaceable>;</option>
+					</term>
+					<listitem><para>
+							Enable pkcs15 emulation (Default:
+							<literal>true</literal>).
+					</para></listitem>
+				</varlistentry>
+				<varlistentry>
+					<term>
+						<option>try_emulation_first = <replaceable>bool</replaceable>;</option>
+					</term>
+					<listitem><para>
+							Prefer pkcs15 emulation code before the normal
+							pkcs15 processing (Default:
+							<literal>no</literal>).  Some cards work in
+							emu-only mode, and do not depend on this
+							option.
+					</para></listitem>
+				</varlistentry>
+				<varlistentry>
+					<term>
+						<option>enable_builtin_emulation = <replaceable>bool</replaceable>;</option>
+					</term>
+					<listitem><para>
+							Enable builtin emulators (Default:
+							<literal>true</literal>).
+					</para></listitem>
+				</varlistentry>
+				<varlistentry>
+					<term>
+						<option>builtin_emulators = <replaceable>emulators</replaceable>;</option>
+					</term>
+					<listitem><para>
+							List of the builtin pkcs15 emulators to test
+							(Default: <literal>esteid, openpgp, tcos,
+								starcert, itacns, infocamere, postecert,
+								actalis, atrust-acos, gemsafeGPK,
+								gemsafeV1, tccardos, PIV-II</literal>)
```

Comment:
This list is probably outdated. There is much more emulators these days including the cac and coolkey.